src/main/java/bio/terra/service/auth/iam/IamProviderInterface.java

```diff
@@ -260,6 +260,17 @@ String getPetToken(AuthenticatedUserRequest userReq, List<String> scopes)
    */
   String getGroup(String accessToken, String groupName) throws InterruptedException;
 
+  List<String> getGroupPolicyEmails(String accessToken, String groupName, String policyName)
+      throws InterruptedException;
+
+  List<String> addGroupPolicyEmail(
+      String accessToken, String groupName, String policyName, String memberEmail)
+      throws InterruptedException;
+
+  List<String> removeGroupPolicyEmail(
+      String accessToken, String groupName, String policyName, String memberEmail)
+      throws InterruptedException;
+
```

Comment:
I think some documentation on these would be great. I like that the add and remove email return the new list of emails in the group, I think that mirrors the other policy member methods, but some documentation explaining what is returned would be helpful.